v03_pipeline/lib/tasks/trigger_hail_backend_reload.py

```diff
@@ -0,0 +1,31 @@
+import luigi
+import luigi.util
+import requests
+
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.model import Env
+from v03_pipeline.lib.tasks import UpdateVariantAnnotationsTableWithNewSamplesTask
+from v03_pipeline.lib.tasks.base.base_project_info_params import (
+    BaseLoadingRunWithProjectInfoParams,
+)
+
+logger = get_logger(__name__)
+
+
+@luigi.util.inherits(BaseLoadingRunWithProjectInfoParams)
+class TriggerHailBackendReload(luigi.Task):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.done = False
+
+    def requires(self):
+        return self.clone(UpdateVariantAnnotationsTableWithNewSamplesTask)
+
+    def run(self):
+        url = f'{Env.HAIL_BACKEND_SERVICE_HOSTNAME}:{Env.HAIL_BACKEND_SERVICE_PORT}/reload_globals'
+        res = requests.post(url, headers={'From': 'loading-pipelines'}, timeout=300)
```

Comment:
This is a great!  Maybe `{'From': 'pipeline-runner'}` for consistency with the helm chart